espnet2/spk/pooling/chn_attn_stat_pooling.py

```diff
@@ -0,0 +1,54 @@
+import torch
+import torch.nn as nn
+
+from espnet2.spk.pooling.abs_pooling import AbsPooling
+
+
+class ChnAttnStatPooling(AbsPooling):
+    """
+    Aggregates frame-level features to single utterance-level feature.
+    Proposed in B.Desplanques et al., "ECAPA-TDNN: Emphasized Channel
+    Attention, Propagation and Aggregation in TDNN Based Speaker Verification"
+
+    """
+
+    def __init__(
+        self,
+        input_size: int = 1536,
+        output_size: int = 3072,
+    ):
```

Comment:
Can you add a docstring here?

Reply:
It is unclear to me how the values of `input_size` and `output_size` should be set.